chore: lock v1 deps (#2591)
Redocly/redocly-cli: .changeset/small-pandas-scream.md

---
"@redocly/respect-core": patch
"@redocly/openapi-core": patch
"@redocly/cli": patch
---

> Pinned runtime deps and key tooling versions for v1 stability.

Reviewer: ```suggestion
Pinned runtime dependency and key tooling versions to improve v1 stability.
```
